staff/miguel-angel-sevillano/CV/index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Miguel Angel Sevillano CV</title>
+</head>
+
+<body>
+    <section>
+        <h1>Personal info</h1>
+        <article>
+            <b>Name:</b> Miguel Angel.<br>
```

Comment:
@MiguiHidalgo `<b>` is style, and `<strong>` is semantic. Please update them all. More info in the following link: https://stackoverflow.com/questions/271743/whats-the-difference-between-b-and-strong-i-and-em